util/gpool/gpool.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gpool
+
+import (
+	"errors"
+	"math"
+	"time"
+)
+
+const (
+	// DefaultPoolSize is the default capacity for a default goroutine pool.
+	DefaultPoolSize = math.MaxInt32
+
+	// DefaultCleanIntervalTime is the interval time to clean up goroutines.
+	DefaultCleanIntervalTime = 5 * time.Second
```

Comment:
Seems this DefaultCleanIntervalTime  is not used.  and also what is this "clean up goroutines" meaning?

Reply:
DefaultCleanIntervalTime is used as the default value of ```ExpiryDuration```. 

Yes, it will remove the useless goroutine.